src/main/java/com/gmail/inverseconduit/BotConfig.java

```diff
@@ -103,6 +123,10 @@ public Path getJavadocsDir() {
         return javadocs;
     }
 
+    public List<Long> getAdmins() {
```

Comment:
I'm aware that the commands already handle this but eventually we might want to watch for privilege escalation socket events so it better be separated for now.